eam/simulator/agency.go

```diff
@@ -153,23 +153,23 @@ func (m *Agency) DestroyAgency(
 
 func (m *Agency) Disable(
 	ctx *simulator.Context,
-	req *types.Disable) soap.HasFault {
+	req *types.Agency_Disable) soap.HasFault {
 
 	m.Runtime.GoalState = string(types.EamObjectRuntimeInfoGoalStateDisabled)
 
-	return &methods.DisableBody{
-		Res: &types.DisableResponse{},
+	return &methods.Agency_DisableBody{
+		Res: &types.Agency_DisableResponse{},
 	}
 }
 
 func (m *Agency) Enable(
```

Comment:
Looks like this needs to be renamed to `Agency_Enable` too, otherwise tests fail with:
```
agency_test.go:226: ServerFaultCode: Agency:deaf259b-3352-4a75-b957-9669de63920f does not implement: Agency_Enable
```